package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "mini-lightbox",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "description": "Minimalist image lightbox",
-  "main": "index.js",
+  "main": "lib/mini-lightbox.js",
```

Comment:
Okay, wait. dist-it is creating a UMD standalone version of a browserify module. Awesome.
Means, you are right, we don't need that much boilerplate. But that also means everyone not using CommonJS _has to_ use the minified file.

If you can live with that, I'll revert some of the changes accordingly.